### PR TITLE
Cm 201 Edit URL Routes 

### PR DESCRIPTION
**Pull request by nataliasabadysh**
### Jira Ticket:
CM-201

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/jira/software/projects/CM/boards/11?assignee=633db6bb61dbef2805c1f25c&selectedIssue=CM-201

### CODE REVIEW

**Review by foley-lynn — APPROVED** [review]
Code looks fine. Some style notes. Please ensure that you prefix every commit with the ticket number. After merging the branch name can get lost, so having the ticket number on each commit can be helpful. Also please rename the PR title to conform to CM-XXX: [description].

Also please don't include two commits with the same message. It's fine to add another commit, but make it specific to the files changed.

Thanks!

File: src/cms/data/menus.json [hunk 1]
```diff
@@ -2308,7 +2308,7 @@
     },
     {
       "title": "Find a Park",
-      "url": "/explore",
+      "url": "/find-a-park",
       "pageType": "Unique",
       "order": 1,
       "parent": "Home",
```
[the remark this answers is not in the document]
Comment (molund) [reply src/cms/data/menus.json:2314]: Menus are loaded during the initial seed, but they won't be re-loaded as part of the data-reimport.  
You will need to manually go into the "Menus" collection type in the Strapi admin interface and update the URLs

[the remark this answers is not in the document]
Comment (foley-lynn) [reply src/cms/data/menus.json:2314]: Yeah, I just noticed that's the old seed data. Wondering if we should pull out all that stuff along with he instructions for seeding, etc..

Regardless, Natalia, could you make sure that you're explicit in the ticket about needing to update the menu manually on each environment. Thanks.

[the remark this answers is not in the document]
Comment (nataliasabadysh) [reply src/cms/data/menus.json:2314]: Sure, I left the comments in the ticket 

https://bcparksdigital.atlassian.net/jira/software/projects/CM/boards/11?assignee=633db6bb61dbef2805c1f25c&selectedIssue=CM-201

**Review by molund — APPROVED** [review]

**Pull request merged**